traffic_ops/testing/api/v4/cachegroups_test.go

```diff
@@ -584,3 +586,67 @@ func CheckCacheGroupsAuthentication(t *testing.T) {
 		t.Error(fmt.Errorf(errFormat, "DeleteCacheGroupByID"))
 	}
 }
+
+func VerifyPaginationSupportCg(t *testing.T) {
+	qparams := url.Values{}
+	qparams.Set("orderby", "id")
+	cachegroup, _, err := TOSession.GetCacheGroupsByQueryParams(qparams)
```

Comment:
Session.GetCacheGroupsByQueryParams() is deprecated, this should use Session.GetCacheGroupsByQueryParamsWithHdr()